aws/scripts/validate-service.sh

```diff
@@ -4,7 +4,7 @@
 sleep 10
 
 # Check the application health status
-response=$(curl --write-out '%{http_code}' --silent --output /dev/null http://localhost:8080/actuator/health)
+response=$(curl --write-out '%{http_code}' --silent --output /dev/null -X POST http://localhost:8080/actuator/shutdown)
```

Comment:
Please, revert any changes from this file. This is not correct. This call will actually shutdown service - so it will "break" out AWS.